Userland/Libraries/LibGfx/Font/OpenType/Font.cpp

```diff
@@ -489,6 +497,14 @@ ErrorOr<NonnullRefPtr<Font>> Font::try_load_from_offset(ReadonlyBytes buffer, u3
     if (opt_kern_slice.has_value())
         kern = TRY(Kern::from_slice(opt_kern_slice.value()));
 
+    Optional<Fpgm> fpgm;
+    if (opt_fpgm_slice.has_value())
+        fpgm = Fpgm(opt_fpgm_slice.value());
```

Comment:
Question: why are these not extracted into functions similar to `Font::font_program()` and `Font::control_value_program()`?

Reply:
I'm not sure what you mean? This is the initial loading of the font where the tables are first extracted, it's only ever run once when the font is loaded. It is a little oddly factored but this is pretty much what the code looks like for all tables, and I didn't want to start refactoring things in the PR. (See the how the OS/2 and Kern tables are loaded just above).